### PR TITLE
code rewritten and phase 2 completed

### PR DESCRIPTION
**Pull request by Saif-Shines**

### CODE REVIEW

File: Freshdesk/v2/simple_events_api_app/app/app.js [hunk 1]
```diff
@@ -2,38 +2,41 @@
...
-    console.log("Failed to get");
-  });
+let showNotification, showError, init, client;
 
+$(document).ready(function () {
+  init();
 });
+
+init = function () {
+  console.log('App init invoked');
```
Comment (ravirajthedeveloper) [inline Freshdesk/v2/simple_events_api_app/app/app.js]: Avoid console logs. Use console.info if the log is necessary to identify the flow and add console.error where the errors need to be printed.

File: Freshdesk/v2/simple_events_api_app/app/app.js [hunk 2]
```diff
@@ -2,38 +2,41 @@
...
+  app.initialized().then(function (_client) {
+    showNotification(_client);
+  }, function (err) {
+    showError(err);
+  });
+}
+
+showNotification = function (clientObj) {
+  client = clientObj;
```
Comment (ravirajthedeveloper) [inline Freshdesk/v2/simple_events_api_app/app/app.js]: The client object can be assigned to the window since it will be used across the app.

**Comment by Saif-Shines** [comment]
@ravirajthedeveloper I made the changes. If you feel they are correct enough, can you please merget his PR ?

File: Freshdesk/v2/simple_events_api_app/app/app.js [hunk 3]
```diff
@@ -2,38 +2,43 @@
...
-    client.interface.trigger("showNotify",{
-      type:"warning",
-      message:{
-        title: "Failed to connect",
-        description:"Error: Your message has not been sent"
-      }
-    });
-    console.log("Failed to get");
-  });
+let showNotification, showError, init, client;
```
Comment (hemchander23) [inline Freshdesk/v2/simple_events_api_app/app/app.js]: Should **let** be used? It is not ES5 compatible.

File: Freshdesk/v2/simple_events_api_app/app/app.js [hunk 4]
```diff
@@ -33,12 +32,11 @@ showNotification = function (clientObj) {
 }
 
 showError = function (err) {
-  console.log("Inside showError method");
   client.interface.trigger("showNotify", {
     type: "warning",
     message: {
       title: "Failed to connect",
-      description: "Error: Your message has not been sent"
+      description: `Error: Your message has not been sent. The error is ${JSON.stringify(err)}`
```
Comment (hemchander23) [inline Freshdesk/v2/simple_events_api_app/app/app.js]: Kindly avoid showing the error object directly. Messages shown to the user has to be in layman's terms. It is fine to log the error object using `console.error()`. However, it is not preferable to show the stringified error message directly. Ignore if it's fixed already
